website/content/index.html

```diff
@@ -137,11 +135,13 @@ <h3>Adapters</h3>
 
 <section class="bf-questions">
     <div class="container">
-            <div class="col-md-12 text-center cta">
-                And much more... Interested? You can check our <a href="https://cwiki.apache.org/confluence/display/GEODE/Index#Index-Geodein5minutesGeodein5minutes" target="_blank" class="btn btn-inverse btn-lg">Geode in 5 minutes tutorial</a> <span class="avoidwrap">, ask a question on the <a href="/community/" class="btn btn-inverse btn-lg">Mailing lists</a> or <a href="http://stackoverflow.com/search?q=Apache%20Geode" class="btn btn-inverse btn-lg">StackOverflow</a></span>
-            </div>
-    </div>
-</section
+		<div class="col-md-12 text-center cta">
+			<h2>Ready to get started?</h2>
+			<p>Follow the tutorial</p>
+			<p><a href="https://cwiki.apache.org/confluence/display/GEODE/Index#Index-Geodein5minutesGeodein5minutes" class="btn btn-inverse btn-lg" target="_blank">Geode in five minutes</a></p>
+		</div>
+	</div>
+</section>
```

Comment:
While I agree http://stackoverflow.com/search?q=Apache%20Geode does not seem super active and it might be fine to drop that reference here, https://geode.apache.org/community might be good to keep?